src/config/configuration.ts

```diff
@@ -1,19 +1,40 @@
+import { z } from 'zod';
+
 import { SupportedChainId } from './constants';
 
-export default () => ({
-  about: {
-    name: 'safe-gelato-relay-service',
-  },
-  applicationPort: process.env.APPLICATION_PORT || '3000',
-  relay: {
-    ttl: process.env.THROTTLE_TTL || 60 * 60, // 1 hour
-    limit: process.env.THROTTLE_LIMIT || 5,
-  },
-  gelato: {
-    apiKey: {
-      [SupportedChainId.GOERLI]: process.env.GELATO_GOERLI_API_KEY,
-      [SupportedChainId.GNOSIS_CHAIN]: process.env.GELATO_GNOSIS_CHAIN_API_KEY,
-    },
-  },
-  gatewayUrl: process.env.GATEWAY_URL || 'https://safe-client.safe.global',
+const ConfigurationSchema = z.object({
+  applicationPort: z.string(),
+  relay: z.object({
+    ttl: z.number(),
+    limit: z.number(),
+  }),
+  gelato: z.object({
+    apiKey: z.object({
+      [SupportedChainId.GOERLI]: z.string(),
```

Comment:
I think this is a wider issue. Would we have separate prod. and staging builds? If so, this could be conditional in the prod. build (we'd need to set a flag in the script for it to reference).